drivers/flash/flash_cadence_qspi_nor_ll.c

```diff
@@ -0,0 +1,997 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <string.h>
+#include "flash_cadence_qspi_nor_ll.h"
+#include <zephyr/logging/log.h>
+
+#define LESS(a, b)   (((a) < (b)) ? (a) : (b))
+#define MORE(a, b)   (((a) > (b)) ? (a) : (b))
```

Comment:
why not use `MIN`/`MAX`? `<zephyr/sys/util.h>`

Reply:
Addressed, Thanks